modules/claims_api/lib/bgs_service/standard_data_service.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module ClaimsApi
+  class StandardDataService < ClaimsApi::LocalBGS
+    def bean_name
+      'StandardDataService'
```

Comment:
Tyler had a similar question on the CorporateUpdateWebService [PR](https://github.com/department-of-veterans-affairs/vets-api/pull/19547): Should this be StandardDataServiceBean/StandardDataService or similar, for consistency with our other services?

Reply:
Au yep I see that now, updating to bring that in line

Reply:
@mchristiansonVA  I feel like the answer to your question here is 'no' but worth making sure

Reply:
Oh, I think what I was overlooking is that they may both be the same  `StandardDataService /StandardDataService`

Reply:
in line with [slack discussion](https://oddball.slack.com/archives/C019XKGS1CN/p1733507742231179) adjusted the `bean_name` [c3ee1a3](https://github.com/department-of-veterans-affairs/vets-api/pull/19620/commits/c3ee1a310cf9049d4edd1a626539578770defd95) 

Good callout @mchristiansonVA to help keep these consistent